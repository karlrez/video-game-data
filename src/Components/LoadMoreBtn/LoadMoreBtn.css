.load-more-btn {
  width: 100%;
  height: 30px;
}
