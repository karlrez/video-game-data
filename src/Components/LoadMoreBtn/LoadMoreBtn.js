import "./LoadMoreBtn.css";

function LoadMoreBtn(props) {
  return (
    <button className="load-more-btn" onClick={props.onClick}>
      Load More
    </button>
  );
}

export default LoadMoreBtn;
