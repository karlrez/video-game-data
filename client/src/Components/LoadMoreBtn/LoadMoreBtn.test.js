import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import LoadMoreBtn from "./LoadMoreBtn";

configure({ adapter: new Adapter() });

let wrapper;
beforeEach(() => {
  wrapper = shallow(<LoadMoreBtn />);
});

afterEach(() => {
  wrapper.unmount();
});

describe("LoadMoreBtn component", () => {
  it("Matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("Correct text", () => {
    expect(wrapper.find(".load-more-btn").text()).toEqual("Load More");
  });
});
