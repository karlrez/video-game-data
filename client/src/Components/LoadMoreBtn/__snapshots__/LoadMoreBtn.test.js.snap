// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadMoreBtn component Matches snapshot 1`] = `ShallowWrapper {}`;
